src/recaptcha.js

```diff
@@ -8,6 +8,8 @@ export default class ReCAPTCHA extends React.Component {
     this.handleExpired = this.handleExpired.bind(this);
   }
 
+  captchaEl = null;
```

Comment:
This line is causing the Travis build to fail.

Is declaring `captchaEl = null` actually needed?